modules/osqp-cpp/0.0.0-20231004-4343373/source.json

```
@@ -0,0 +1,10 @@
+{
+    "url": "https://github.com/google/osqp-cpp/archive/43433736334d6b515ea4b0247156fea9e56c0d3f.tar.gz",
+    "integrity": "sha256-Ye1aTRuJMM0T6u6Z/qZLAt8iCEzmRArQVxOUPAHkwSQ=",
+    "strip_prefix": "osqp-cpp-43433736334d6b515ea4b0247156fea9e56c0d3f",
+    "patches": {
+        "add-build-file.patch": "sha256-h4DBMwvSkV+caG2l/KGwBnrrM0UcbdgjEr2Dn9YTZTw=",
+        "add_module_dot_bazel.patch": "sha256-dv+i2dhf/c1zvvR8NWKFMKUSzWVjJdEt5ZoZ8QLtf2s="
+    },
+    "patch_strip": 1
```

Remove this as your patches do not have a prefix
```suggestion
    }
```
```suggestion
    },
    "patch_strip": 1
```